addons/mail/models/mail_thread.py

```diff
@@ -2274,6 +2274,28 @@ def _message_compute_parent_id(self, parent_id):
                 parent_id = current_ancestor.parent_id.id if current_ancestor.parent_id else parent_id
         return parent_id
 
+    def _message_compute_subject(self, record_names=None, model_description=None):
+        """ Helper to get the default subject when creating a message on this thread
```

Comment:
Seems indent could be improved a bit ? + I generally add a void line between the docstring, the parameters and the return, to ease reding of docstrings :)